sdk/eventgrid/eventgrid/swagger/README.md

```diff
@@ -18,6 +18,8 @@ save-inputs: true
 source-code-folder-path: ./src/generated
 typescript: true
 hide-clients: true
+use-extension:
+  "@autorest/typescript": "6.0.0-beta.4"
```

Comment:
Starting with `-beta.5` the code generator started to replace some `any`'s with `Record<string, unknown>` for places where an opaque object was taken.  For now I've just locked to the slightly earlier version, so I don't have to ingest these changes right now.

Reply:
I noticed something similar when re-generating Tables. It looks like if the Swagger defines `additionalProperties` on an object, M4 assumes AnyObject instead of `any`. In the Tables swagger we had `additionalProperties: true` and as a workaround, I set the following transform which makes M4 interpret `additionalProperties` as `any`

```yml
directive:
  - from: swagger-document
    where: $.definitions.TableEntityProperties
    transform: >
      $["additionalProperties"] = {};
```